src/Compilers/Test/Core/Compilation/TestOperationVisitor.cs

```diff
@@ -735,6 +735,11 @@ public override void VisitUnaryOperator(IUnaryOperation operation)
 
             AssertConstrainedToType(operatorMethod, operation.ConstrainedToType);
             Assert.Same(operation.Operand, operation.ChildOperations.Single());
+
+            // Directly get the symbol for this operator from the semantic model.  This allows us to exercise
+            // potentially creating synthesized intrinsic operators.
+            if (operation.SemanticModel != null)
+                operation.SemanticModel.GetSymbolInfo(operation.Syntax);
```

Comment:
>operation.SemanticModel.GetSymbolInfo(operation.Syntax); [](http://example.com/codeflow?start=16&length=56)

Since we no longer assert anything in constructors, do we still want to add this in this PR, given that the other PR will have to make validation stronger anyway? Same question for the other change in this file.